README.md

```diff
@@ -49,7 +55,21 @@ Additionally, in case not all the container is clickable (i.e., does not produce
 </div>
 ```
 
+Finally, adding further information to purchases can be made by passing the `ts-data-items` JSON array:
+
+```html
+<div
+  data-ts-action="purchase"
+  data-ts-items='[{"product": "product-id-purchase-1", "quantity":1, "price": 2399}, {"product": "product-id-purchase-2", "quantity": 2, "price": 399}]'
+>
+  My purchase
+</div>
+```
+
+Either mix quotes (single/double) or escape them inside the string.
```

Comment:
I'd mention how they need to escape them. 

Also, this section also applies to productIds in case they somehow have products with characters that need to be escaped (&, ', ", <,>). Although maybe we should not allow to pass productIds, and rely just on resolvedBidIds. That basically means we won't be receiving organic events, which we don't use at all.